src/aap_eda/settings/features.py

```diff
@@ -0,0 +1,37 @@
+#  Copyright 2025 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from typing import Any
+
+from django.conf import settings
+from flags.state import flag_enabled
+
+DISPATCHERD: bool
+ANALYTICS: bool
+
+_feature_flag_cache = {}
```

Comment:
Suggestion: no need to implement custom cache, you can use https://docs.python.org/3/library/functools.html#functools.cache